src/banana.js

```diff
@@ -32,16 +35,17 @@ module.exports = function bananaChecker( dir, options, logErr ) {
 		requireLowerCase: true,
 		requireMetadata: true,
 
+		allowLeadingWhitespace: true,
+		allowTrailingWhitespace: true,
+
 		skipIncompleteMessageDocumentation: []
 	}, options );
 
 	const jsonFilenameRegex = /(.*)\.json$/;
+	const leadingWhitespaceRegex = /^\s+/;
```

Comment:
Micro-optimisation: The + in unnecessary. Detecting one character of whitespace is sufficient.